Contest_API.md

```diff
@@ -534,6 +528,33 @@ event feed, also note that these should not be considered proper REST
 endpoints, and that the `event-feed` endpoint is a streaming feed in
 NDJSON format.
 
+Note that `access`, `account`, `state`, `scoreboard` and `event-feed`
+are singular nouns and indeed contain only a single object.
+
+### Required and Optional API
+
+The only required endpoints are metadata: `api` and `access`. The
+only required property is `id` (if it exists on the endpoint).
```

Comment:
The claim about properties is technically true, but I find it confusing. I would like to improve or remove that sentence.

Reply:
How? I couldn't get past the fact that it is indeed true.

Reply:
Fair point :).

Could we maybe say something like "The only requirements for properties is that collections must have the `id` property", and maybe also "all endpoints that are supported must have at least some property, i.e. an endpoint can't be empty."

Reply:
Updated to this. It felt a little odd to have the referential integrity lower down, so I moved it to this paragraph as well. Please review.